schema/table_test.go

```diff
@@ -92,6 +92,7 @@ func TestTable(t *testing.T) {
 		type Perms struct {
 			View   bool
 			Create bool
+			Model  *Model
```

Comment:
This is just a regular (JSON_ field, e.g. `Foo sql.NullBool` is just the same. Why do we need to treat it differently?

Reply:
Also, https://github.com/uptrace/bun/issues/1125 is about embedded structs, but this is not. `Perms` is not even a model...

Reply:
Fixed is #1137